base/server/src/main/java/org/dogtagpki/server/rest/base/AuditServletBase.java

```diff
@@ -47,6 +47,7 @@ public class AuditServletBase {
 
     public AuditServletBase(CMSEngine engine, String username) {
         this.engine = engine;
+        this.userName = username;
     }
```

Comment:
Yes, I have follow this approach in all the services because the initial design, without annotation, was differently organised. I think there is room for this optimisation in several services but not all since in same cases the initialisation requires information provided by the request (e.g. `SecurityDomainServletBase` require the `locale`) and need additional work.
I'll do this optimisation in separate PR.